src/codewhisperer/util/runtimeLanguageContext.ts

```diff
@@ -3,103 +3,147 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { getLogger } from '../../shared/logger/logger'
 import { CodewhispererLanguage } from '../../shared/telemetry/telemetry.gen'
```

Comment:
This `CodewhispererLanguage` is actually telemetry language? can we rename it to be that, otherwise it's still confusing

Reply:
yes, just need to update the name in common repo, but not sure if rename will cause any concern

Reply:
We can alias the import like `import { CodewhispererLanguage as CodewhispererTelemetryLanguage } from ...` but I don't know how many places we are importing this type so just an idea